src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -1,13 +1,16 @@
 ﻿using System;
 using System.Threading.Tasks;
 using IdentityServer4.Events;
+using IdentityServer4.Models;
 using IdentityServer4.Services;
 using IdentityServer4.Stores;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Identity.Api.Exceptions;
 using NHSD.BuyingCatalogue.Identity.Api.Infrastructure;
 using NHSD.BuyingCatalogue.Identity.Api.Models;
 using NHSD.BuyingCatalogue.Identity.Api.ViewModels;
+using SignInResult = Microsoft.AspNetCore.Identity.SignInResult;
```

Comment:
I think this can be removed if you change the variable to use var instead of the actual type?

Reply:
I removed the `var` due to our decision to prefer explicit types where it's not obvious but I'll reinstate `var` here.